src/app/containers/Caption/index.jsx

```diff
@@ -25,11 +26,13 @@ const chooseOffscreenText = (
       return defaultText;
   }
 };
-const renderParagraph = paragraphBlock => (
-  <p key={deepGet([0, 'model', 'text'], paragraphBlock)}>
-    <Blocks blocks={paragraphBlock} componentsToRender={componentsToRender} />
-  </p>
-);
+const renderParagraph = paragraphBlock => {
+  return (
```

Comment:
Why was this updated?

Reply:
@sareh suggested it in case of punctuation in the key